src/cljs/athens/views/devtool.cljs

```diff
@@ -1,394 +0,0 @@
-(ns athens.views.devtool
-  (:require
-    ["/components/Icons/Icons" :refer [XmarkIcon ChevronLeftIcon]]
-    ["@chakra-ui/react" :refer [Box IconButton Button Table Thead Tbody Th Tr Td Input ButtonGroup]]
-    [athens.config :as config]
-    [athens.db :as db :refer [dsdb]]
-    [cljs.pprint :as pp]
-    [clojure.core.protocols :as core-p]
-    [clojure.datafy :refer [nav datafy]]
-    [datascript.core :as d]
-    [datascript.db]
-    [me.tonsky.persistent-sorted-set]
-    [re-frame.core :refer [subscribe dispatch]]
-    [reagent.core :as r]
-    [reagent.ratom]
-    [sci.core :as sci])
```

Comment:
Can you remove this dependency from `deps.edn` as well please?

Reply:
Yes